ipn/ipnstate/ipnstate.go

```diff
@@ -90,7 +91,7 @@ type PeerStatus struct {
 	RxBytes       int64
 	TxBytes       int64
 	Created       time.Time // time registered with tailcontrol
-	LastWrite     time.Time // time last packet sent
+	LastWrite     mono.Time // time last packet sent
```

Comment:
How does this serialize in JSON in `tailscale status --json` ?

Reply:
Badly. I've changed it to format as a time.Time, for backwards compatibility. Of course, if the wall clock changes, it won't be responsive to that change.